story/ep1/pt/umi1_8.txt

```diff
@@ -899,15 +899,15 @@
 `{n}Pouco tempo atrás, quando Shannon entrou no quarto, George disse que havia esquecido algo na mansão e precisava voltar para pegar.`
 `Shannon disse que iria acompanhá‐lo, assim como ele fez no caminho para a casa de hóspedes, e os dois partiram juntos.`
 `"......Eles já estavam mostrando sinais disso há algum tempo, sabia?`
-` Perguntando o que o outro gostava, perguntando sobre seus hobbies.`
+` Perguntando o que o outro gostava, perguntando sobre seus passatempos.`
 ` Eu sempre soube que isso era muito mais do que um interesse passageiro!"`
 `"Pensando bem, eu tenho a sensação de que o George‐aniki sempre foi legal demais com a Shannon‐chan......`
 ` Agooora eu entendi..."`
-`"De acordo com a previsão do tempo, parece que a chuva vai ficar pior esta noite.`
+`"Segundo a previsão do tempo, parece que a chuva vai ficar pior esta noite.`
 ` E parece também que ela não vai parar amanhã, mas deve ficar um pouco melhor."`
 `"...Mesmo?`
 ` Então, talvez o barco só chegue depois de amanhã......`
-` ...Estou preocupada que isso possa interferir no seu trabalho segunda‐feira..."`
+` ...Estou preocupada de que isso possa interferir no seu trabalho segunda‐feira..."`
```

Comment:
is really necessary?

Reply:
Na minha opinião soa melhor, mas eu posso reverter